src/universal_transfer_operator/data_providers/filesystem/aws/s3.py

```diff
@@ -148,12 +144,12 @@ def s3_extra_args(self) -> Any:
         return self.dataset.extra.get("s3_extra_args", {})
 
     @property
-    def bucket_name(self) -> str:
+    def bucket_name(self) -> Any:
```

Comment:
Why is this any?

Reply:
Ya, I'm not sure why this is happening, earlier I resolved this by adding `--follow-imports`, but it's still causing an issue.  I have created a separate issue for the same. We will have to address this in phase 2. 
issue: https://github.com/astronomer/apache-airflow-provider-transfers/issues/61